src/components/Button/Button.tsx

```diff
@@ -179,6 +179,15 @@ export function buttonFrom(
       onClick={onAction}
       {...action}
       {...overrides}
+      disabled={false}
```

Comment:
@romesh87 Could you add default value in props params. What If I need to set default value of disable=true from external props?
const button = ({
  disabled = false,
}: Props)

Reply:
@CaptainSadaf these boolean props are not passed to THEMEDBUTTON component by default on the master branch